### ISSUE TITLE
Cannot get data only (only payload including the path)

### ISSUE THREAD

**Issue by laudaget** [matteocrippa/flutter-nfc-reader#43]
Is there a way to get the data without the path provided in payload? Couldn't figure out a way to cut away the path.

The content on my nfc chips are like 100001, 100002, 100003.. I would like to get '1000001' as my response, but the payload gives me 'en100001' instead.

Thank you in advance

**Comment by henrysachs** [comment]
yeah the first 3 bytes need to be cut away on android to do this.

**Comment by sayhicoelho** [comment]
same issue here!

any solution?
